velox/exec/HashJoinBridge.cpp

```diff
@@ -198,4 +191,9 @@ uint64_t HashJoinMemoryReclaimer::reclaim(
       });
   return reclaimedBytes;
 }
+
+bool isHashBuildMemoryPool(const memory::MemoryPool& pool) {
+  static const std::string re(".*HashBuild");
+  return RE2::FullMatch(pool.name(), re);
```

Comment:
Is just ends-with check? If so, no need to use RE2 for that, right?

Reply:
Updated. Thanks!